src/StreamJsonRpc.Desktop/PublicAPI.Shipped.txt

```diff
@@ -67,4 +67,4 @@ static StreamJsonRpc.JsonRpc.Attach(System.IO.Stream sendingStream, System.IO.St
 static StreamJsonRpc.JsonRpc.Attach(System.IO.Stream stream, object target = null) -> StreamJsonRpc.JsonRpc
 virtual StreamJsonRpc.DelimitedMessageHandler.Dispose(bool disposing) -> void
 virtual StreamJsonRpc.JsonRpc.Dispose(bool disposing) -> void
-virtual StreamJsonRpc.JsonRpc.InvokeCoreAsync<ReturnType>(int? id, string targetName, System.Collections.Generic.IReadOnlyList<object> arguments, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<ReturnType>
```

Comment:
This is actually an edit, since I had to add the "isParameterObject" parameter (see next change).  Are you saying I shouldn't break this API?